api/managment/views.py

```diff
@@ -29,16 +38,20 @@ def list(self, request, *args, **kwargs):
         user = request.user
         patient = Patient.objects.filter(user=user)
         guardian = Guardian.objects.filter(user=user)
-        res = dict()
-        if patient.exists():
-            res["patient"] = PatientSerializer(patient.first()).data
-
-        if guardian.exists():
-            res["guardian"] = GuardianSerializer(guardian.first()).data
-
-        res["user"] = UserSerializer(user).data
-
-        res["bought"] = True
+        res = {"type": None, "user": None}
+        if len(patient) != 0:
+            res["type"] = "patient"
+            res["user"] = PatientSerializer(tuple(patient)[0]).data
+        elif len(guardian) != 0:
+            res["type"] = "guardian"
+            res["user"] = GuardianSerializer(tuple(guardian)[0]).data
+        else:
+            res["type"] = "nothing"
+            res["user"] = UserSerializer(user).data
```

Comment:
Revert as in the master
![image](https://user-images.githubusercontent.com/61160614/235427156-453d734d-f29e-47b0-b5bd-5a30f4325acb.png)